api/openapi-spec/v3/api__v1_openapi.json

```diff
@@ -1425,26 +1425,26 @@
               }
             ],
             "default": {},
-            "description": "Details about the container's last termination condition."
+            "description": "LastTerminationState holds the last termination state of the container to simplify debugging of container crashes and restarts. This field if not populated if container is still running and RestartCount is 0."
```

Comment:
Same changes here too.

Reply:
`make update` failed me, rerunning! Thank you for catching